.travis.yml

```diff
@@ -9,11 +9,14 @@ matrix:
 before_install:
   # PRs made to 'master' branch should always orginate from another repo or the 'dev' branch
   - '[ $TRAVIS_PULL_REQUEST = "false" ] || [ $TRAVIS_BRANCH != "master" ] || ([ $TRAVIS_PULL_REQUEST_SLUG = $TRAVIS_REPO_SLUG ] && [ $TRAVIS_PULL_REQUEST_BRANCH = "dev" ])'
-  # Pull the docker image first so the test doesn't wait for this
-  - docker pull nfcore/rnafusion
-  # Fake the tag locally so that the pipeline runs properly
-  - docker tag nfcore/rnafusion nfcore/rnafusion:latest
-  # Pull the docker image first so the test doesn't wait for this
+  - if [ $TRAVIS_BRANCH == "master" ]; then 
+      docker pull nfcore/rnafusion;
```

Comment:
Shouldn't you pull nfcore/rnafusion:latest each time?

Reply:
By default if the tag is not specified, latest will be pulled. I've just copy pasted the required lines by `nf-core lint`:
ERROR: Test Failures:
  http://nf-co.re/errors#5: CI is not pulling the correct docker image. Should be:
    'docker pull nfcore/rnafusion'
  http://nf-co.re/errors#5: CI is not tagging docker image correctly. Should be:
    'docker tag nfcore/rnafusion nfcore/rnafusion:latest'

Reply:
After discussion, this doesn't solve the problem. Dropping down to
```
- docker pull nfcore/rnafusion:dev
- docker tag nfcore/rnafusion:dev nfcore/rnafusion:latest
- docker pull nfcore/test-datasets:rnafusion
```